src/structures/Guild.js

```diff
@@ -1041,6 +1041,8 @@ class Guild {
    * @property {PermissionResolvable} [denied] The permissions to deny
    * **(deprecated)**
    * @property {GuildMemberResolvable|RoleResolvable} memberOrRole Member or role this overwrite is for
```

Comment:
Looks like the property was incorrectly renamed in #2734 (https://github.com/discordjs/discord.js/commit/3021e5ce7fc5208585ff4bac417198619a31e562#diff-e5c54069adfa0d32480eb3cc9faeee20L979)

Just changing it back would fix this, as `memberOrRole` does not (and never did) exist as a valid property here.

```suggestion
    * @property {GuildMemberResolvable|RoleResolvable} id Member or role this overwrite is for
```

The lines below would need to go too.